src/illumidesk/illumidesk/apis/nbgrader_service.py

```diff
@@ -18,24 +16,23 @@
 nbgrader_db_port = os.environ.get("POSTGRES_NBGRADER_PORT") or 5432
 nbgrader_db_password = os.environ.get("POSTGRES_NBGRADER_PASSWORD")
 nbgrader_db_user = os.environ.get("POSTGRES_NBGRADER_USER")
+nbgrader_db_name = os.environ.get("POSTGRES_NBGRADER_DATABASE")
```

Comment:
If database does not exist, I am getting the following error 

`[W 2022-04-01 23:08:55.292 JupyterHub base:1256] Rolling back session due to database error (psycopg2.OperationalError) FATAL:  database "tea_course" does not exist`